tests/basic/async_dr_test.go

```diff
@@ -149,6 +153,135 @@ var _ = Describe("{MigrateDeployment}", func() {
 	})
 })
 
+var _ = Describe("{MigrateDeploymentMetroAsync}", func() {
```

Comment:
Can we use ValidateApplications after switching to the second cluster to see if the apps are up and running?

Reply:
We will implement that validation method soon in upcoming phase, as we have to add deactivation on source and activation on destination.